roles/databases/influxdb/templates/influxdb_external_service.yaml.j2

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: indluxdb-external
```

Comment:
Shouldn't it be for internal access only? A clusterIP?

Reply:
I don't know much about k8s, if the question doesn't make sense, disregard the question.

Reply:
That is right. External access should be for testing only, sorry.

Reply:
That was changed.